detox/specs/elements/Pages.ts

```diff
@@ -0,0 +1,49 @@
+import { by, element, waitFor } from "detox";
+
+export function Pages() {
+    const clickOnButton = async (buttonName: string) => {
+        const button = element(by.id(buttonName));
+        await waitFor(button)
+            .toExist()
+            .withTimeout(2000);
+        await button.tap();
+    };
+    return {
+        async openActivityIndicator() {
+            await clickOnButton("actionButton15");
```

Comment:
I wish these button names ("actionButtons15") were not gone through random number generator 🍡 This will be hard for later to actually maintain

Reply:
Maybe olga can do this ?

Reply:
Names of widget are generated on create, but could be changes, in the widget props > common > Name